packages/headless/src/controllers/facets/automatic-facet-generator/headless-automatic-facet-generator.ts

```diff
@@ -53,6 +52,48 @@ export interface AutomaticFacetGeneratorProps {
   options: AutomaticFacetGeneratorOptions;
 }
 
+/**
+ * A scoped and simplified part of the headless state that is relevant to the `Automatic Facet` controller.
+ */
+export interface AutomaticFacetState {
+  /**
+   * The automatic facet field.
+   */
+  field: string;
+  /**
+   * The automatic facet label.
+   */
+  label: string;
+  /**
+   * The automatic facet values.
+   */
+  values: FacetValue[];
+}
+/**
+ * The `AutomaticFacet` controller allows you to create a search interface component that the end user
+ * can use to refine a query by selecting filters based on item metadata (i.e., field values). Unlike regular facets that
```

Comment:
```suggestion
 * can interact with to refine a query by selecting filters based on item metadata (i.e., field values). Unlike regular facets that
```

pet peeve, but I just don't like "that the end _user_ can _use_" 😅